core/domain/collection_domain_test.py

```diff
@@ -136,13 +144,13 @@ def test_nodes_validation(self):
         self.collection.nodes = [
             collection_domain.CollectionNode.from_dict({
                 'exploration_id': '0',
-                'prerequisite_skills': [],
-                'acquired_skills': ['skill0a']
+                'prerequisite_skill_ids': [],
+                'acquired_skill_ids': ['skill0a']
```

Comment:
Is this right? I thought all skill IDs should be of the form ``skill####``? This doesn't fail validation errors?